### ISSUE TITLE
STRETCH: add a spanish logo in instruction screen (as per anywhere ballot design)

### ISSUE THREAD

**Issue by beausmith** [votingworks/bmd#47]

**Comment by beausmith** [comment]
@benadida - can you add a bit more clarity here? maybe link to thing you are describing? Did you mean to ask for a "choose language" screen?

**Comment by benadida** [comment]
Yes, either a language choice screen or language toggle option in an existing screen at the beginning that presents one other language as an option (e.g. spanish), even if we don't have it implemented yet. Let's see how that screen looks.
